### DISCUSSION TITLE
Make rpm or deb package for fio-plot

### DISCUSSION THREAD

**Discussion by zhucan** [louwrentius/fio-plot#120] [Ideas]
It is more convenient installation and use.

**Comment by louwrentius** [comment]
Hello, fio-plot is available through pip (pypi), isn't that good enough? 
I'm always keeping the pypi version up-to-date.

(Try: pip install fio-plot)

**Reply by zhucan** [reply]
not enough, beacause we always test under Intranet environment，The external network is unreachable， we need to download it firstly at local.

**Comment by zhucan** [comment]
only need to run "python setup.py bdist_rpm" can make the rpm package. And use the [fpm](https://github.com/jordansissel/fpm) can convert the rpm to deb.

**Comment by louwrentius** [comment]
I understand the use case. Sorry for being a broken record, but if I build an rpm and deb package, I have to test them too and at this time I have no need for this. In this case if there would be instructions to do a git clone and run a script that generates the rpm or deb that's ok, as long as users know it's not something that is tested. Maybe github actions can be used, but I have no experience with this. Same thing here, if you want to share something I'll integrate this with the mentioned caveat, but I understand if you don't want to do that.
